example2/.gitignore

```diff
@@ -0,0 +1,4 @@
+.gradle
+build
+ts-frontend/node_modules
```

Comment:
@saberduck Do we need this exclusion? We already have `node_modules` in top level `.gitignore`